packages/covector/__snapshots__/index.test.ts.snap

```diff
@@ -1514,6 +1514,721 @@ Object {
+              "@types/fs-extra": "9.0.1",
+              "@types/http-proxy": "1.17.4",
+              "@types/imagemin": "7.0.0",
+              "@types/imagemin-optipng": "5.2.0",
+              "@types/jsdom": "16.2.3",
+              "@types/lodash": "4.14.155",
+              "@types/ms": "0.7.31",
+              "@types/sharp": "0.25.0",
+              "@types/webpack-merge": "4.1.5",
+              "@typescript-eslint/eslint-plugin": "3.1.0",
+              "@typescript-eslint/parser": "3.1.0",
+              "babel-jest": "26.0.1",
+              "dotenv": "8.2.0",
+              "eslint": "7.1.0",
+              "eslint-config-standard-with-typescript": "18.0.2",
+              "eslint-plugin-import": "2.20.2",
+              "eslint-plugin-lodash-template": "0.18.0",
+              "eslint-plugin-node": "11.1.0",
+              "eslint-plugin-promise": "4.2.1",
+              "eslint-plugin-security": "1.4.0",
+              "eslint-plugin-standard": "4.0.1",
+              "husky": "4.2.5",
+              "is-running": "2.1.0",
+              "jest": "26.0.1",
+              "jest-mock-process": "1.4.0",
+              "lint-staged": "10.2.8",
+              "lockfile-lint": "4.3.6",
+              "promise": "8.1.0",
+              "raw-loader": "4.0.1",
+              "ts-loader": "7.0.5",
+              "typescript": "3.9.3",
+              "webpack": "4.43.0",
+              "webpack-cli": "3.3.11",
+              "webpack-node-externals": "1.7.2",
+            },
+            "funding": Object {
+              "type": "opencollective",
+              "url": "https://opencollective.com/tauri",
+            },
+            "homepage": "https://github.com/tauri-apps/tauri#readme",
+            "husky": Object {
+              "hooks": Object {
+                "pre-commit": "lint-staged",
+              },
+            },
+            "license": "MIT",
+            "lint-staged": Array [
+              "eslint --fix",
+            ],
+            "name": "tauri",
+            "repository": Object {
+              "type": "git",
+              "url": "git+https://github.com/tauri-apps/tauri.git",
+            },
+            "scripts": Object {
+              "build": "webpack --progress",
+              "build-release": "npm run build --display none --progress false",
+              "build:tauri[rust]": "cd ../tauri && TAURI_DIST_DIR=../../test/fixture/dist TAURI_DIR=../test/fixture cargo publish --dry-run --allow-dirty",
+              "lint": "eslint --ext ts ./src/**/*.ts",
+              "lint-fix": "eslint --fix --ext ts ./src/**/*.ts",
+              "prepublishOnly": "npm run build-release",
+              "pretest": "npm run build",
+              "test": "jest --runInBand --no-cache --testPathIgnorePatterns=\\"(build|dev)\\"",
+              "test:local": "jest --runInBand",
+            },
+            "version": "0.6.2-branch-name.12345",
+          },
+          "version": "0.6.2-branch-name.12345",
```

Comment:
Looks like we are getting the version here 👍 . It doesn't seem like it is attempting to publish it though, I imagine because of the change files? May be worth creating a new fixture if it is painful finding one that works for your tests.

Reply:
I was looking for something like `"Checking if tauri.js@0.6.2-branch-name.12345 is already published with: echo 0.5.2",`, but not seeing it which is kind of confusing. Like I don't see any of the output mention trying to publish `tauri.js`.

Reply:
But yea, you are right, it does say `"tauri.js [publish]: echo publishing tauri.js would happen here",`. I wonder if it is worth checking the version there by adjusting that publish command to echo it out?

Reply:
So in that fixture, there's a change file for tauri and tauri-updater. tauri-updater isn't in the snapshot because publish is set to false. tauri.js gets a preview bump because it lists tauri as a dependency. But that doesn't explain why tauri isn't getting bumped and why tauri.js isn't trying to publish. This might have to do with the bug I pointed out about the interdependent bumping logic.